receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -270,6 +270,11 @@ func (k *K8sAPIServer) init() error {
 
 	go k.startLeaderElection(ctx, lock)
 
+	// TODO: this is example code for running the scrape client.  It needs to be protected by leader electiongit
```

Comment:
nit
```suggestion
	// TODO: this is example code for running the scrape client.  It needs to be protected by leader election
```
Think you typed `git` here by accident

Reply:
good find! I will remove this in a follow up PR (or rev if necessary)